src/main/resources/application.yml

```diff
@@ -89,3 +89,8 @@ management:
     info:
       cache:
         time-to-live: 2000ms
+
+notify:
+  enabled: true
+  templates:
+    licenceApproved: 36c3f270-7393-4595-96f7-db2033cccd91
```

Comment:
```suggestion
    licenceApproved: ${NOTIFY_TEMPLATES_LICENCEAPPROVED}
```

Reply:
I left a default value in the application.yaml (for template IDs) is so that it will default to this when running locally, without having to set an environment variable and would be overridden by values from Helm in real environments - dev, preprod and prod.